Quantum-Control-Applications/Superconducting/Single Fixed Transmon/allxy.py

```diff
@@ -303,56 +304,66 @@
 
     with stream_processing():
         for i in range(21):
-            I_st[i].boolean_to_int().average().save(f"I{i}")
-            Q_st[i].boolean_to_int().average().save(f"Q{i}")
+            I_st[i].average().save(f"I{i}")
+            Q_st[i].average().save(f"Q{i}")
 
```

Comment:
You sure?

Reply:
yes, the measurement statement in this allxy does a simple measurement not state discrimination